pkg/pkg.ml

```diff
@@ -0,0 +1,8 @@
+#use "topfind"
+#require "topkg-jbuilder"
+
```

Comment:
it doesnt detect cstruct as the package name, possibly because ppx_cstruct breaks the rule of everything starting with `cstruct-` -- hence I set it manually. cc @dim